Shoko.Server/API/v3/Models/Shoko/WebUI.cs

```diff
@@ -217,6 +217,14 @@ public WebUISeriesFileSummary(SVR_AnimeSeries series, HashSet<EpisodeType> episo
                     if (file?.Media == null || location == null)
                         return null;
 
+                    return new { file, xref, location };
```

Comment:
replace with `return (file, xref, location);` to get deconstructing working (again).

Reply:
```suggestion
                    return (file, xref, location);
```